### PR TITLE
docs: add troubleshooting to medusa-file-s3 plugin

### LINKED ISSUE #1943: Docs: S3 Documentation troubleshooting
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

When installing the `medusa-file-s3` plugin there may be instances (experienced by [at least 2 users in discord](https://discord.com/channels/876835651130097704/877195433649258536/1003273089284571156) ) where fiels are not uploaded correctly and the backend displays the error: `AccessControlListNotSupported: The bucket does not allow ACLs`

### How can this issue be resolved?

This can be resolved by adding a `Troubleshooting` section to the plugin documentation with the following from https://stackoverflow.com/questions/70333681/for-an-amazon-s3-bucket-deplolyent-from-guithub-how-do-i-fix-the-error-accesscon

1. go to the `Bucket -> Permissions` tab 
2. scroll to `Object Ownership` and click `edit`
3. select `ACLs enabled` and `Bucket owner preferred`
4. click the `Save Changes`

### Are you interested in working on this issue?

- [X] I would like to fix this issue

### PR DESCRIPTION
**Pull request by richardwardza**
Closes #1943 

This updates the documentation to add some troubleshooting to the medusa-file-s3 plugin documentation.

The fix has been show to work with 2 installations so far - context is here: https://discord.com/channels/876835651130097704/877195433649258536/1003273089284571156

### CODE REVIEW

File: docs/content/add-plugins/s3.md [hunk 1]
```diff
@@ -87,3 +87,17 @@ The two access keys in the options are the ones created in the previous section.
 ### Try it out
 
 Finally, run your Medusa server alongside our admin system to try out your new file service. Upon editing or creating products, you can now upload thumbnails and images, that are stored in an AWS S3 bucket.
+
+### Troubleshooting
```
Comment (shahednasser) [inline docs/content/add-plugins/s3.md:91]:
```suggestion
### Troubleshoot Errors
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Just a small change to the wording 🙂

File: docs/content/add-plugins/s3.md [hunk 2]
```diff
@@ -87,3 +87,17 @@ The two access keys in the options are the ones created in the previous section.
...
+
+ #### The bucket does not allow ACLs
+
+ If the backend file upload fails with this message then you need to enable ACLs on your S3 bucket.
+
+ To do this, 
+ - go to the `Bucket -> Permissions` tab 
+ - scroll to `Object Ownership` and click `edit`
+ - select `ACLs enabled` and `Bucket owner preferred`
+ - click the `Save Changes`
```
Comment (shahednasser) [inline docs/content/add-plugins/s3.md:101]: Can you capitalize the first letter of the first word of each item in the list?

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: docs/content/add-plugins/s3.md (hunk 2 above)
Comment (richardwardza) [inline docs/content/add-plugins/s3.md:101]:
```suggestion
 - Go to the `Bucket -> Permissions` tab 
 - Scroll to `Object Ownership` and click `edit`
 - Select `ACLs enabled` and `Bucket owner preferred`
 - Click the `Save Changes`
```

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks! @richardwardza

**Pull request merged**